### PR TITLE
Added some inspectable to work with arrow direction, background color…

### PR DESCRIPTION
**Pull request by thihaaung6245**
Added New Inspectable

- Arrow Direction (Upside or down)
- Arrow Background Color
- Arrow Width
- Arrow Height

### CODE REVIEW

File: Sources/ACTabScrollView.swift [hunk 1]
```diff
@@ -173,7 +182,11 @@ open class ACTabScrollView: UIView, UIScrollViewDelegate {
         seperatorView.backgroundColor = textColor
         
         // arrow
-        arrowView.frame.origin = CGPoint(x: (self.frame.width - arrowView.frame.width) / 2, y: tabSectionHeight)
+        if upsideArrowDirection{
```
Comment (azurechen) [inline Sources/ACTabScrollView.swift:185]: according to the swift guideline, there should be one more space between upsideArrowDirection and {

File: Sources/ACTabScrollView.swift [hunk 2]
```diff
@@ -173,7 +182,11 @@ open class ACTabScrollView: UIView, UIScrollViewDelegate {
...
+            arrowView.frame.origin = CGPoint(x: (self.bounds.width - arrowView.bounds.width) / 2, y: tabSectionHeight - arrowView.bounds.height)
+        }else{
```
Comment (azurechen) [inline Sources/ACTabScrollView.swift:187]: same issue

File: Sources/ACTabScrollView.swift [hunk 3]
```diff
@@ -385,7 +398,13 @@ open class ACTabScrollView: UIView, UIScrollViewDelegate {
             contentSectionScrollView.frame = CGRect(x: 0, y: tabSectionHeight, width: self.frame.size.width, height: contentSectionHeight)
             
             // reset the origin of arrow view
-            arrowView.frame.origin = CGPoint(x: (self.frame.width - arrowView.frame.width) / 2, y: tabSectionHeight)
+
```
Comment (azurechen) [inline Sources/ACTabScrollView.swift:401]: remove the unnecessary newline

File: Sources/ACTabScrollView.swift [hunk 4]
```diff
@@ -548,28 +567,42 @@ class ArrowView : UIView {
...
+            ctx.addQuadCurve(to: CGPoint(x:maxX * 0.2 , y: maxY * 0.2), control: CGPoint(x: maxX * 0.12, y: 0))
+            ctx.addLine(to: CGPoint(x: midX - maxX * 0.05, y: maxY * 0.9))
+            ctx.addQuadCurve(to: CGPoint(x: midX + maxX * 0.05, y: maxY * 0.9), control: CGPoint(x: midX, y: maxY))
+            ctx.addLine(to: CGPoint(x: maxX * 0.8, y: maxY * 0.2))
+            ctx.addQuadCurve(to: CGPoint(x: maxX, y: 0), control: CGPoint(x: maxX * 0.88, y: 0))
+            ctx.closePath()
+            
+            ctx.setFillColor((arrorBackgroundColor?.cgColor)!)
+            ctx.fillPath();
+        }
```
Comment (azurechen) [inline Sources/ACTabScrollView.swift:605]:
```
ctx.beginPath()

if upsideArrowDirection! {
    ctx.move(to: CGPoint(x: 0, y: 1))
    ctx.addQuadCurve(to: CGPoint(x: maxX * 0.2 , y: maxY * 0.8), control: CGPoint(x: maxX * 0.12, y: 1))
    ctx.addLine(to: CGPoint(x: midX - maxX * 0.05, y: maxY * 0.1))
    ctx.addQuadCurve(to: CGPoint(x: midX + maxX * 0.05, y: maxY * 0.1), control: CGPoint(x: midX, y: maxY))
    ctx.addLine(to: CGPoint(x: maxX * 0.8, y: maxY * 0.8))
    ctx.addQuadCurve(to: CGPoint(x: maxX, y: 1), control: CGPoint(x: maxX * 0.88, y: 0))
} else {
    ctx.move(to: CGPoint(x: 0, y: 0))
    ctx.addQuadCurve(to: CGPoint(x: maxX * 0.2 , y: maxY * 0.2), control: CGPoint(x: maxX * 0.12, y: 0))
    ctx.addLine(to: CGPoint(x: midX - maxX * 0.05, y: maxY * 0.9))
    ctx.addQuadCurve(to: CGPoint(x: midX + maxX * 0.05, y: maxY * 0.9), control: CGPoint(x: midX, y: maxY))
    ctx.addLine(to: CGPoint(x: maxX * 0.8, y: maxY * 0.2))
    ctx.addQuadCurve(to: CGPoint(x: maxX, y: 0), control: CGPoint(x: maxX * 0.88, y: 0))
}

ctx.closePath()
ctx.setFillColor((arrorBackgroundColor?.cgColor)!)
ctx.fillPath();
```

File: Sources/ACTabScrollView.swift [hunk 5]
```diff
@@ -24,13 +28,17 @@ open class ACTabScrollView: UIView, UIScrollViewDelegate {
...
     @IBInspectable open var tabGradient: Bool = true
-    @IBInspectable open var arrowIndicator: Bool = false
     @IBInspectable open var pagingEnabled: Bool = true {
         didSet {
             contentSectionScrollView.isPagingEnabled = pagingEnabled
         }
     }
     @IBInspectable open var cachedPageLimit: Int = 3
+    @IBInspectable open var arrowIndicator: Bool = false
+    @IBInspectable open var upsideArrowDirection : Bool = true
```
Comment (azurechen) [inline Sources/ACTabScrollView.swift:38]: please set default false for the demo at readme file.

**Review by azurechen — CHANGES_REQUESTED** [review]
Please reset the following files. These files shouldn't be changed in this commit.
```
.DS_Store
ACTabScrollView.xcodeproj/project.pbxproj
ACTabScrollView/Base.lproj/Main.storyboard
ACTabScrollView/Info.plist
```

File: Sources/ACTabScrollView.swift (hunk 4 above)
Comment (thihaaung6245) [reply Sources/ACTabScrollView.swift:605]: Close this pull request now cause I am having trouble marge and conflict issue

**Pull request closed without merging**